Lombiq.HelpfulExtensions/ResourceManagementOptionsConfiguration.cs

```diff
@@ -12,7 +12,7 @@ public class ResourceManagementOptionsConfiguration : IConfigureOptions<Resource
     static ResourceManagementOptionsConfiguration() =>
         _manifest
             .DefineScript(TargetBlank)
-            .SetUrl(WwwRoot + "scripts/target-blank.min.js", WwwRoot + "scripts/target-blank.js")
+            .SetUrl(WwwRoot + "js/target-blank.min.js", WwwRoot + "js/target-blank.js")
```

Comment:
I would argue that as long as we don't need, or expect to need in the near future, to reuse the `WwwRoot` path, prefer one that removes even more duplication, e.g.:
```suggestion
            .SetUrl(ScriptRoot + "target-blank.min.js", ScriptRoot + "target-blank.js")
```
Replace `Wwwroot` with `ScriptRoot`, and add the `js/` part to it.